glusterfs/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "Red Hat Gluster Storage",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "GlusterFS",
+  "metric_prefix": "glusterfs.",
+  "metric_to_check": "glusterfs.cluster.nodes.count",
+  "creates_events": false,
+  "short_description": "Monitor GlusterFS cluster node, volume, and brick status metrics.",
+  "guid": "1cb9a21c-8cc4-4727-a4b1-ab7015c7ae24",
+  "support": "core",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Red Hat Gluster Storage",
+  "categories": [
+    "data store"
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "red-hat-gluster-storage",
```

Comment:
Should ID be glusterfs or the branding vendor name? The logos have the vendored name.